site/en/docs/extensions/mv3/content_scripts/index.md

```diff
@@ -663,6 +655,49 @@ inspected by the content script and then posted to the extension process. In thi
 establishes a line of communication to the extension process. The reverse is possible through
 similar means.
 
+## Accessing extension files {: #files }
+ 
+To access an extension file from a content script, you can call
+[`chrome.runtime.getURL()`][api-get-url] to get the _absolute URL_ of your extension asset as shown in the following example (`content.js`):
```

Comment:
I included the file's name for the code sample in parentheses because I was instructed not to use the [labels component](https://developer.chrome.com/docs/handbook/components/#labels) until the formatting is fixed.